lib/Service/FileService.php

```diff
@@ -509,7 +512,6 @@ public function formatFiles(array $files, ?array $requestParams = []): array
 		}
 
 		// @todo search
-		$total   = count($formattedFiles);
```

Comment:
because formattedFiles can fail?

Reply:
No, formattedFiles is already cut to the size of the limit, and will never contain the total amount of files